UPGRADE-3.x.md

```diff
@@ -1,6 +1,16 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.21 to 3.22
```

Comment:
You can pick ideas from here: https://github.com/sonata-project/SonataAdminBundle/pull/6212/files#diff-dcb51da20ad02d4fde6c4b5fb83819d0R4-R10

I think it is good to have similar upgrade notes